### PR TITLE
feat: add container to multiple text elements

### LINKED ISSUE #6418: support wrapping multiple text elements into container at once
The context menu (right click) action of wrapping a text element in a container should work when multiple text elements selected.

I'd also spread out the element if necessary to make them fit without overlap if containerizing would make them not to.

### PR DESCRIPTION
**Pull request by ChinmayMhatre**
Closes #6418

Updated `actionCreateContainerFromText` action to take multiple text elements.
Iterated over each element and adding containers to them

https://user-images.githubusercontent.com/51131670/230577307-4a6a56af-94c1-4ec5-ac6a-9fa515fea2f7.mov

### CODE REVIEW

File: src/actions/actionBoundText.tsx [hunk 1]
```diff
@@ -185,107 +185,111 @@ export const actionCreateContainerFromText = register({
...
-          if (endBinding?.elementId === textElement.id) {
-            endBinding = { ...endBinding, elementId: container.id };
-          }
-
-          if (startBinding || endBinding) {
-            mutateElement(ele, { startBinding, endBinding });
-          }
+    let updatedElements = elements.slice();
+    let updatedAppState = appState;
+    let containerIds:any = {};
```
Comment (dwelle) [inline src/actions/actionBoundText.tsx:198]: let's use the `AppState["selectedElementIds"]` type here

**Comment by dwelle** [comment]
Looks good @ChinmayMhatre!

In the original issue I suggested spreading the elements as well to ensure they don't overlap after containerizing, but it may not be so simple. I'll create a new issue for it and we can do it separately.

**Comment by ChinmayMhatre** [comment]
> Looks good @ChinmayMhatre!
> 
> In the original issue I suggested spreading the elements as well to ensure they don't overlap after containerizing, but it may not be so simple. I'll create a new issue for it and we can do it separately.

Yes, it could come in another issue. I gave it a try but couldn't get it to work so I thought of pushing just this commit first.

**Comment by ChinmayMhatre** [comment]
> Looks good @ChinmayMhatre!
> 
> In the original issue I suggested spreading the elements as well to ensure they don't overlap after containerizing, but it may not be so simple. I'll create a new issue for it and we can do it separately.

Yes we could do that ! I have updated the type and fixed some linting warnings. Let me know if anything else is needed!

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
Thanks for the PR, we'll ship soon ❤️

**Comment by Fran5ois** [comment]
Thank you very much for implementing this!

There seems to be a limitation: 

When one of the text elements selected is already wrapped in a container, the action doesn't show anymore in the context menu...

I opened an [issue ](https://github.com/excalidraw/excalidraw/issues/9334) for this...

**Pull request merged**